include/clap/ext/draft/note-expressions.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include "../../plugin.h"
+
+// This extension reports which note expressions are supported by a plugin.
+// If this extension isn't present all note expressions are assumed to be supported.
+
+// TODO : is the clap_supported_note_expressions bitmask future-proof? Or should it be a "count / get-info" pattern?
+// TODO : does getNumMIDIChannels still make sense in a 'real' extension? It can probably removed if we go for a "count / get-info" pattern?
+

```

Comment:
Get num midi Chan doesn’t belong here. That’s an oddity of vst3 needing params for midi cc and we can leave it in the wrapper